examples/with-vue-thirdpartyemailpassword/src/views/AuthView.vue

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import SuperTokensReactComponent from "../components/Supertokens";
+
+export default{
+    beforeMount(){
+        this.render()
+    },
+    beforeUpdate(){
+        this.render()
+    },
+    mounted(){
+        console.log('mounted')
```

Comment:
> console.log('mounted')

You should remove all console logs

Reply:
Have fixed this in the updated PR.

Reply:
there is one more console.log have added a comment